### PR TITLE
[Feat][Cats] - Cat Integration

### PR DESCRIPTION
**Pull request by dandalandone**
## Tickets
- link ticket here

## Changes
- Add Cat API
- Add Cat controller, service, and routing

## Notes
- No database integration

### CODE REVIEW

File: backend/src/cats/cats.service.ts
Comment (TreshAnn) [inline backend/src/cats/cats.service.ts:1]: please add doraemon

**Review by markjoshuacarmen — APPROVED** [review]
lgtm

**Review by sandrex-endino — APPROVED** [review]
LGTM

**Review by KezTimple — APPROVED** [review]

**Review by sandrex-SGV — CHANGES_REQUESTED** [review]
Please add another cat in the array

File: backend/src/cats/cats.service.ts
Comment (TreshAnn) [reply backend/src/cats/cats.service.ts:1]: please add other cats

Comment (TreshAnn) [inline backend/src/cats/cats.service.ts:1]: please add doraemon

**Review by TreshAnn — CHANGES_REQUESTED** [review]
please check request changes

File: backend/src/cats/cats.service.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const getCats = () => {
+    return ['Miming', 'Garfield'];
```
Comment (TreshAnn) [inline backend/src/cats/cats.service.ts]:
```suggestion
    return ['Miming', 'Garfield', 'Doraemon', 'puspin'];
```

**Review by TreshAnn — CHANGES_REQUESTED** [review]
Please check another changes.

File: backend/src/cats/cats.service.ts (hunk 1 above)
Comment (dandalandone) [reply backend/src/cats/cats.service.ts]: Request changes done, please recheck! @TreshAnn

**Review by sandrex-SGV — APPROVED** [review]

**Review by TreshAnn — APPROVED** [review]

**Review by TreshAnn — APPROVED** [review]

**Review by sandrex-endino — APPROVED** [review]
LGTm

**Pull request merged**